dds_web/config.py

```diff
@@ -35,7 +35,6 @@ class Config(object):
     LOCAL_TEMP_CACHE = "/dds_web/local_temp_cache"
     DDS_S3_CONFIG = "/code/dds_web/sensitive/s3_config.json"
     SAFESPRING_URL = os.environ.get("DDS_SAFESPRING_URL", "https://example.endpoint.net")
-    DDS_SAFESPRING_PROJECT = os.environ.get("DDS_SAFESPRING_PROJECT", "dds.example.com")
```

Comment:
I think there might be a few other places that needs updating for this as well? The README and the db_init perhaps? Should we even remove `safespring_name` column for Units?

Reply:
No that should be kept for invoicing purposes 💰

Reply:
But other places aside from the column, yes